df.job-types.xml

```diff
@@ -449,6 +449,7 @@
             <item-attr name='caption' value='Construct Blocks'/>
             <item-attr name='type' value='Manufacture'/>
             <item-attr name='item' value='BLOCKS'/>
+            <item-attr name='skill' value='CUT_STONE'/>
```

Comment:
Remark: only STONE blocks will use that skill, while blocks made of other materials will use different skills.

Reply:
That's fair. Should I leave this one untagged, then? We didn't have a good way to associate multiple values, do we?